docs/community/release_notes/release_note_0.4.0.md

```diff
@@ -0,0 +1,268 @@
+---
+hide:
+  - navigation
+
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Apache Celeborn(Incubating) 0.4.0 Release Notes
+
+## Highlight
+
+- Rerun Spark Stage for Celeborn Shuffle Fetch Failure
+- Added support for Hadoop MapReduce
+- Added support for Flink 1.18
```

Comment:
let's respect the Apache trademark
```suggestion
- Added support for Apache Hadoop MapReduce
- Added support for Apache Flink 1.18
```
Also, the site needs to update
<img width="1437" alt="image" src="https://github.com/apache/incubator-celeborn-website/assets/26535726/23e223a1-83db-4994-8a51-3007addc54b8">

Reply:
would you help to update the website code? @waitinfuture